Joker/src/components/tab-bar.tsx

```diff
@@ -0,0 +1,55 @@
+// import {RouteProp} from '@react-navigation/native';
+import {BottomTabNavigationEventMap} from '@react-navigation/bottom-tabs';
+import {
+  NavigationHelpers,
+  ParamListBase,
+  TabNavigationState,
+} from '@react-navigation/native';
+import React, {useCallback} from 'react';
+import {StyleSheet, Text, TouchableOpacity, View} from 'react-native';
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
+import {IS_IOS} from '../variables';
+
+type TabbarProps = {
+  navigation: NavigationHelpers<ParamListBase, BottomTabNavigationEventMap>;
+  state: TabNavigationState<ParamListBase>;
+};
+export function TabBar(props: TabbarProps) {
+  const tabRoutes = props.state.routeNames;
+  const index = props.state.index;
+  const insets = useSafeAreaInsets();
+  const onPressTab = useCallback(
+    (route: string) => {
+      props.navigation.navigate(route);
+    },
+    [props.navigation],
+  );
+
+  return (
+    <View style={[styles.wrapper, {paddingBottom: Math.max(insets.bottom)}]}>
```

Comment:
This construction doesn't make sense right now
```js
Math.max(insets.bottom)
```
[See this](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Math/max)